src/components/Form.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import Button from './Button';
+
+export default function Form({ onAddActivity }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target;
+    const { activity, friend } = form.elements;
+    onAddActivity({ activity: activity.value, friend: friend.value });
```

Comment:
hey jelena! really nice work so far...i´m not sure but if you want to update your state with onAddActivity you should also spread the things in there which were in there before!?

Reply:
for e.g. from react beta`setArtists( // Replace the state
  [ // with a new array
    ...artists, // that contains all the old items
    { id: nextId++, name: name } // and one new item at the end
  ]
);`

Reply:
Thank you for the good eye on this.😊
I have the function in the app component, thats what you mean right? 
` function onAddActivity({ activity, friend }) {
    setActivities([...activities, { activity, friend }]);
  }`

Reply:
👍